http/src/main/java/com/redhat/lightblue/client/http/request/LightblueHttpRequest.java

```diff
@@ -1,9 +0,0 @@
-package com.redhat.lightblue.client.http.request;
-
-import org.apache.http.client.methods.HttpRequestBase;
-
-public interface LightblueHttpRequest {
```

Comment:
A `LightblueRequest` is and always was an HTTP request, so there was no need for a separate `LightblueHttpRequest` AFAIK. The only thing `LightblueRequest` was missing was the HTTP method, but it had the URI (path params) and request body. I suppose request body could be used to talk to lightblue another way other than HTTP if we really wanted to be that generic but, not sure what URI would be for if not over HTTP, so yeah, that's why I consolidated those interfaces.